content/github/setting-up-and-managing-your-github-user-account/adding-an-email-address-to-your-github-account.md

```diff
@@ -1,4 +1,4 @@
----
+sylwiapanekpanorama@gmail.com ---
```

Comment:
...ging-your-github-user-account/adding-an-email-address-to-your-github-account.md